community-fund/COMMUNITY_FUND_POLICY.md

```diff
@@ -0,0 +1,79 @@
+# Cross Project Council Community Fund
+
+## About The Community Fund
+
+The OpenJS Foundation’s community fund supports travel and attendance at events and activities directly related to OpenJS projects, technologies, or goals. We support travel for individuals who are:
+
+* Collaborators
+* Standards body participants
+* Speakers
+
+The fund can cover the costs, in whole or in part, of a participant's hotel, airfare, and some event registrations.
+Applicants should explore employer-sponsored funding options as their first source of financial support.
+Additionally, 25% of the community fund is allocated specifically to support diverse participants.
+Funding decisions are determined by a combination of need and potential impact.
+
+## Community Fund Rules
+
+Valid travel expenses are:
+
+* Coach airfare tickets
+* Accommodations (Dates of the event plus one night prior are eligible for reimbursement)
+* Ground transportation to/from the airport
+
+NOTE: Funds may not be used for miscellaneous travel expenses including food, visa costs, non-airport transportation, baggage fees, etc.
+Generally, registration can be reimbursed if the attendee is speaking on behalf of OpenJS, the CPC, or OpenJS-hosted projects.
+Funding is not transferable to another person, or any other event. 
+
+Travel fund assistance can only be received once per year.
```

Comment:
This seems like an odd restriction, presumably if someone used the fund to go to a summit then they would not be able to request it again for standard body participation, etc. which could end up hurting what this fund is for. I think a limit based on the total amount of funds requested per year per person makes more sense, or this can be capped at a value higher than 1 (maybe 2 or 3).

Reply:
Agreed. Either put a cap on the max yearly allocation per person as a % of the overall budget or a $ amount or just add a more open statement such as "Funding will be allocated fairly so that no single individual benefits excessively from it." (I'd favor the later which clarifies the intention, and leaves useful wiggle room.)

Reply:
Removed line pending discussion in https://github.com/openjs-foundation/cross-project-council/issues/1239